api/definitions/v1.buildlog/description.adoc

```diff
@@ -0,0 +1 @@
+A BuildLog is the (unused) resource associated with the build log redirector for retrieving build logs from Kubernetes. This is not an API object anyone is expected to use and only serves an internal purpose.
```

Comment:
> A BuildLog is the (unused) resource associated with the build log redirector for retrieving build logs from Kubernetes.  

A BuildLog is used to document the output of the REST call to retrieve the logs of a build. It is not a concrete type and is used only for documentation.